packages/ui/client/composables/module-graph.ts

```diff
@@ -12,7 +12,7 @@ export type ModuleGraphConfig = GraphConfig<ModuleType, ModuleNode, ModuleLink>
 function defineExternalModuleNode(module: string): ModuleNode {
   let label = module
   if (label.includes('/node_modules/'))
-    label = label.split(/\/node_modules\//g).pop()!.split(/\//g).shift()!
+    label = label.split(/\/node_modules\//g).pop()!
```

Comment:
Will it show the full name? Like `@org/something/dist/esm/index.esm.js`? Maybe we should just detect `@org` and show `@org/something` in this case?

Although if we have something like `package/node` and `package/client`, i still will see only `package` 🤔 Hm, I wonder if there is a way to show only what was imported 🤔 I think we always get the full path 😞

Reply:
It would be better if it could be expressed in short form, such as `@org/somethingA` and `@org/somethingB`. I will modify the code.

Reply:
I read the code and found that it is unlikely to directly determine the appropriate short form based on module name.

Reply:
Yeah, let's do that!